Replace informal usage of hyphens with more appropriate formatting (#7376)
ppy/osu-wiki: wiki/Tournaments/GTS/AGTS_2020/en.md

   - If there are 5 players or less signed up for a country, those players will form the team representing their country.
   - In case there are more than 10 players signed up for a country, players will decide amnong themselves who will partake in the tournament. If they cannot decide the top 10 players, the staff will choose it sorted by ranking to form the 2 teams.
   - If there are 6 players, players will have to decide on the 5 players who will partake in the tournament. If they cannot decide the top 5 players, the staff will choose it sorted by ranking to form the team.

>    - If there are players between 7—10 players (after keeping the best ranked ones) signed up for a country, the players from that country will have a few days to decide how to split themselves between the 2 teams. It's recommended to come to an agreement as fast as possible to then decide on the captain and be able to schedule and prepare their matches rapidly.

Author: ```suggestion
   - If there are players between 7–10 players (after keeping the best ranked ones) signed up for a country, the players from that country will have a few days to decide how to split themselves between the 2 teams. It's recommended to come to an agreement as fast as possible to then decide on the captain and be able to schedule and prepare their matches rapidly.
```

>    - If there are players between 7–10 players (after keeping the best ranked ones) signed up for a country, the players from that country will have a few days to decide how to split themselves between the 2 teams. It's recommended to come to an agreement as fast as possible to then decide on the captain and be able to schedule and prepare their matches rapidly.

Reviewer: [no en dash here](https://www.thepunctuationguide.com/en-dash.html)

```suggestion
   - If there are players between 7 to 10 players (after keeping the best ranked ones) signed up for a country, the players from that country will have a few days to decide how to split themselves between the 2 teams. It's recommended to come to an agreement as fast as possible to then decide on the captain and be able to schedule and prepare their matches rapidly.
```

4. In the Group Stage, all teams from each group will face each other. The **best 2 teams** get our of their group.
5. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
   1. Most matches won.

>    2. Highest (number of beatmaps won—number of maps lost) score.

Reviewer: the hyphen acts as a minus sign here

Author: ```suggestion
   2. Highest (number of beatmaps won - number of maps lost) score.
```

5. The mappool will be made by a special team of mappoolers.
6. The event will be a 4 vs 4 match with the following rules:
   - Best-of-15 format (First to 8 points).

>    - Mappool will consist of 20 FreeMod maps and 1 TB. All the maps can be—played with any mods combination. Some of them might be impossible to—put a mod on though!

Reviewer: ???

```suggestion
   - Mappool will consist of 20 FreeMod maps and 1 TB. All the maps can be played with any mod combinations. Some of them might be impossible to put a mod on though!
```
